client/src/app/site/deployment-center/deployment-center-setup/wizard-logic/deployment-center-state-manager.ts

```diff
@@ -71,15 +74,17 @@ export class DeploymentCenterStateManager implements OnDestroy {
         const siteDescriptor = new ArmSiteDescriptor(this._resourceId);
         return forkJoin(
           siteService.getSite(this._resourceId),
+          siteService.getSiteConfig(siteDescriptor.getSiteOnlyResourceId()),
           this._cacheService.getArm(`/subscriptions/${siteDescriptor.subscription}`, false, ARMApiVersions.armApiVersion)
         );
       })
       .switchMap(result => {
-        const [site, sub] = result;
+        const [site, siteConfig, sub] = result;
```

Comment:
Suggestion: See if you can distinguish between site & siteConfig to indicate that they might correspond to different site/slot. You can rename siteConfig to webAppConfig or something similar.